src/Eurofurence.App.Server.Web/Controllers/ArtistsAlleyController.cs

```diff
@@ -90,6 +138,16 @@ [EnsureNotNull][FromRoute] Guid id
         [HttpPost("TableRegistrationRequest")]
         public async Task<ActionResult> PostTableRegistrationRequestAsync([EnsureNotNull][FromForm] TableRegistrationRequest request, [Required] IFormFile requestImageFile)
         {
+            if (GetArtistAlleyStatus() == ArtistAlleySystemStatus.REG_CLOSED)
+            {
+                return StatusCode(403, "Table registrations are temporally closed");
```

Comment:
"Your Artist Alley registration cannot be processed at this time. Please contact the Dealers' Den team about your Artist Alley registration."